docs/running_clusterfuzzlite.md

```diff
@@ -178,6 +178,7 @@ Next: choose the [subguide](#subguides) for your CI system.
 ## Supported Continous Integration Systems {#subguides}
 
 - [GitHub Actions]
+- [GitLab]
```

Comment:
This link doesn't go anywhere. Add it to line 192.

Reply:
indeed